build.gradle

```diff
@@ -52,7 +52,7 @@ buildscript {
 apply plugin: "org.ajoberstar.grgit"
 apply plugin: 'maven-publish'
 
-project.version = "1.4.4-SNAPSHOT"
+project.version = "1.4.5-SNAPSHOT"
```

Comment:
Yes, it will be 1.5.0, but using 1.4.5-SNAPSHOT here is still fine.  The snapshot is really just the best guess as to the next version.  We decide which number to increment based on semver rules.  In this case, since we've added some functionality (not just fixed existing functional) and changed no existing functionality, we'll increase the minor version.